vignettes/spatial_wwinference.Rmd

```diff
@@ -0,0 +1,2457 @@
+    linetype = "dashed"
+  ) +
+  xlab("") +
+  ylab("Global Rt Results") +
+  facet_wrap(~gen_model_type, scales = "free_y") +
+  guides(
+    fill = guide_legend(
+      title = "Assumed Corr. Structure"
+    ),
+    color = guide_legend(
+      title = "Assumed Corr. Structure"
+    )
+  ) +
+  scale_fill_manual(
+    breaks = c("Exponential", "Unstructured", "IID"),
+    values = c("darkviolet", "deeppink3", "darksalmon")
+  ) +
+  scale_colour_manual(
+    breaks = c("Exponential", "Unstructured", "IID"),
+    values = c("darkviolet", "deeppink3", "darksalmon")
+  ) +
+  scale_y_continuous(trans = "log10") +
+  theme_bw()
+# ------------------------------------------------------------------------------
+# Site Rt results --------------------------------------------------------------
+site_rt_ribbon_data <- all_pred_draws_df %>%
+  filter(
+    name == "subpopulation R(t)"
+  ) %>%
+  group_by(
+    date,
+    subpop,
+    inf_model_type,
+    gen_model_type
+  ) %>%
+  summarise(
+    lower = quantile(pred_value, 0.025, na.rm = TRUE),
+    median = median(pred_value),
+    upper = quantile(pred_value, 0.975, na.rm = TRUE),
+    .groups = "drop"
+  ) %>%
+  mutate(
+    subpop = sub(
+      pattern = "Site: (\\d+)",
+      replacement = "Site \\1",
+      x = subpop,
+      ignore.case = "remainder of pop"
+    )
+  ) %>%
+  mutate(
+    subpop = case_when(
+      subpop == "remainder of pop" ~ "Aux",
+      .default = subpop
+    )
+  )
+site_rt_result_plot <- ggplot() +
+  geom_ribbon(
+    data = site_rt_ribbon_data,
+    aes(x = date, ymin = lower, ymax = upper, fill = inf_model_type),
+    alpha = 0.2
+  ) +
+  geom_line(
+    data = site_rt_ribbon_data,
+    aes(x = date, y = median, color = inf_model_type),
+    size = 1.25,
+  ) +
+  geom_hline(
+    yintercept = 1,
+    color = "blue",
+    alpha = .7,
+    size = 1,
+    linetype = "dashed"
+  ) +
+  geom_line(
+    data = site_rt_data_full %>%
+      mutate(gen_model_type = model_type),
+    aes(x = date, y = value),
+    size = 1.25
+  ) +
+  geom_vline(
+    xintercept = lubridate::ymd(forecast_date),
+    linetype = "dashed"
+  ) +
+  xlab("") +
+  ylab("Site Rt") +
+  facet_grid(subpop ~ gen_model_type, scales = "free_y") +
+  guides(
+    fill = guide_legend(
+      title = "Assumed Corr. Structure"
+    ),
+    color = guide_legend(
+      title = "Assumed Corr. Structure"
+    )
+  ) +
+  scale_fill_manual(
+    breaks = c("Exponential", "Unstructured", "IID"),
+    values = c("darkviolet", "deeppink3", "darksalmon")
+  ) +
+  scale_colour_manual(
+    breaks = c("Exponential", "Unstructured", "IID"),
+    values = c("darkviolet", "deeppink3", "darksalmon")
+  ) +
+  scale_y_continuous(trans = "log10") +
+  theme_bw()
+# ------------------------------------------------------------------------------
+
+global_rt_result_plot / site_rt_result_plot +
+  plot_layout(
+    guides = "collect",
+    heights = c(1, n_sites)
+  ) &
+  theme(legend.position = "bottom")
```

Comment:
The fit to the aux site looks pretty off in the random correlation matrix. 

what percent fo the population is covered by wastewater data in this study? Just feel like that should be more identifiable from the trend in hospital admissions. Perhaps the prior on the subpopulation R(t) deviations is actually too tight since it seems to be inferring something very close to 1 for all data generating models.